src/styles/block-style/rating-style.scss

```diff
@@ -0,0 +1,30 @@
+.reviews {
+  position: relative;
+  display: flex;
+  justify-content: start;
+  align-items: center;
+
+  margin-bottom: 24px;
+  width: 166px;
+
+  &__stars {
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+
+    &:nth-child(-n + 4) {
```

Comment:
Could you consider keeping functional for this block from the previous task? Currently, modifiers don't work

Reply:
Not fixed